cpp/src/bindings.cpp

```diff
@@ -592,4 +592,6 @@ PYBIND11_MODULE(_pypowsybl, m) {
 
     m.def("get_glsk_factors_end_timestamp", &pypowsybl::getInjectionFactorEndTimestamp, "Get glsk end timestamp", py::arg("importer"));
 
+    m.def("create_feeder_bay", &pypowsybl::createFeederBay, "Create feeder bay", py::arg("network"), py::arg("busbarSectionId"), py::arg("injectionPositionOrder"), py::arg("injectionDirection"), py::arg("throwException"), py::arg("reporter"), py::arg("dataframe"), py::arg("elementType"));
```

Comment:
for python conventions busbar_section_id, injection_position_order ... would be better